docs/specs/git.yml

```diff
@@ -459,8 +459,6 @@ repos:
           Osmond creates this
 inputs:
   docfx.yml: |
-    secrets:
```

Comment:
We should cover the two cases in spec test:
- token from secrets
- token from API call.